src/Cassandra.IntegrationTests/TestClusterManagement/SimulacronManager.cs

```diff
@@ -30,10 +30,7 @@ public static Uri BaseAddress
 
         public void Start()
         {
-            if (_started)
-            {
-                return;
-            }
+            Stop();
```

Comment:
Without going much into detail, its weird to have a `Start()` method that calls a `Stop()` method...

Can we move simulacron fixes and exception test fixes out of this pull request? I think the changes are not directly related.